create_environment.bat

```diff
@@ -32,6 +32,15 @@ if not exist qt-everywhere-opensource-src-4.8.7 (
   exit /b 1
 )
 
+if not exist qt-everywhere-src-5.10.1 (
```

Comment:
Would it be helpful to put the Qt version number in a variable since it's referenced a number of times in this file?